content/_data/changelogs/lts.yml

```
@@ -9733,6 +9733,494 @@
+    category: bug
+    issue: 65368
+    authors:
+      - sunweisheng
+    pr_title: "Remoting agent.jar does not work behind proxy"
+    message: |-
+      Add proxy support for Remoting.
+  - type: major bug
+    category: regression
+    pull: 8564
+    issue: 71937
+    authors:
+      - Vlatombe
+    pr_title: "[JENKINS-71937] Fix deprecated 'Slave' constructor"
+    message: |-
+      Fix agent allocation due to label issue detected by vSphere Cloud plugin (regression in 2.421).
+  - type: major bug
+    category: regression
+    pull: 8613
+    issue: 72189
+    authors:
+      - mawinter69
+    pr_title: "[JENKINS-72189] fix drag&drop handle for existing repeatables"
+    message: |-
+      Fix drag and drop handle for existing repeatables (regression in 2.335).
+  - type: rfe
+    category: rfe
+    pull: 8564
+    issue: 72248
+    authors:
+      - daniel-beck
+    pr_title: "Add telemetry for Jenkins uptime"
+    message: |-
+      Add telemetry for Jenkins uptime.
+  - type: rfe
+    category: rfe
+    pull: 8587
+    issue: 72156
+    authors:
+    - olamy
+    pr_title: "[JENKINS-72156] Backport Upgrade to Winstone 6.14 which contains an upgrade to Jetty 10.0.17"
+    references:
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-6.13
+        title: Winstone 6.13 changelog
+      - url: https://github.com/jenkinsci/winstone/releases/tag/winstone-6.14
+        title: Winstone 6.14 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-10.0.16
+        title: Jetty 10.0.16 changelog
+      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-10.0.17
+        title: Jetty 10.0.17 changelog
+      - url: https://nvd.nist.gov/vuln/detail/CVE-2023-44487
+        title: CVE-2023-44487
+    message: |-
+      Upgrade Winstone from 6.12 to 6.14.
+      This includes the upgrade of Jetty from 10.0.15 to 10.0.17.
+      The Jetty upgrade includes fixes for several CVEs.
+  - type: bug
+    category: regression
+    pull: 8606
+    issue: 72179
+    authors:
+      - daniel-beck
+    pr_title: "[JENKINS-72179] Show description of boolean build parameter values"
+    message: |-
+      Show the description of boolean build parameter values on the Parameters view (regression in 2.179).
+  - type: bug
+    category: regression
+    pull: 8492
+    issue: 72020
+    authors:
+      - Vlatombe
+    pr_title: "[JENKINS-72020] Restore ability to reorder clouds"
+    message: |-
+      Allow clouds to be reordered.
+      This was previously possible, but disappeared when the cloud management was moved to a separate page (regression in 2.403).
+  - type: bug
+    category: regression
+    pull: 8674
+    issue: 70994
+    authors:
+      - basil
+    pr_title: "[JENKINS-70994] Upgrade bundled SnakeYAML plugin from 1.33-95.va_b_a_e3e47b_fa_4 to 2.2-111.vc6598e30cc65"
+    message: |-
+      Update SnakeYAML plugin to 2.0 to silence security scanners (regression in 2.431).
```

Thanks!  It wasn't a regression, just an update to silence security scanners.[  The bundled plugin is now 2.2 rather than 1.33.
```suggestion
      Update SnakeYAML plugin to 2.2 to silence security scanners.
```